cmflib/cmf.py

```diff
@@ -56,20 +57,22 @@ class Cmf:
             pipeline.  If a pipeline already exist with the same name, the existing pipeline object is reused.
         custom_properties: Additional properties of the pipeline that needs to be stored.
         graph: If set to true, the libray also stores the relationships in the provided graph database. The following
-            environment variables should be set: `NEO4J_URI` (graph server URI), `NEO4J_USER_NAME` (user name) and
-            `NEO4J_PASSWD` (user password), e.g.:
-            ```bash
-            export NEO4J_URI="bolt://ip:port"
-            export NEO4J_USER_NAME=neo4j
-            export NEO4J_PASSWD=neo4j
+            variables should be set: `neo4j_uri` (graph server URI), `neo4j_user` (user name) and
+            `neo4j_password` (user password), e.g.:
+            ```
+               cmf init local --path /home/user/local-storage --git-remote-url https://github.com/XXX/exprepo.git --neo4j-user neo4j --neo4j-password neo4j 
+                              --neo4j-uri bolt://localhost:7687
             ```
     """
 
     # pylint: disable=too-many-instance-attributes
-
-    __neo4j_uri = os.getenv('NEO4J_URI', "")
-    __neo4j_user = os.getenv('NEO4J_USER_NAME', "")
-    __neo4j_password = os.getenv('NEO4J_PASSWD', "")
+    # Reading CONFIG_FILE variable
+    cmf_config = os.environ.get("CONFIG_FILE",".cmfconfig")

```

Comment:
Where are we setting this environment variable ?

Reply:
Add documentation for CONFIG_FILE env variable

Reply:
Will do.